docs/04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enable Kyma with JetStream
+---
+
+This guide shows how to enable JetStream and how it upholds `at least once delivery` quality of service.
```

Comment:
```suggestion
This guide shows how to enable JetStream and how it ensures the `at least once` delivery.
```
According to the official [JetStream documentation](https://docs.nats.io/nats-concepts/jetstream#exactly-once-message-delivery), only the 'at least once' wording is distinguished in brackets